config/rollup.config.js

```diff
@@ -0,0 +1,78 @@
+import path from 'path';
+
+import makePlugins from './plugins/index';
+import makeDevServerPlugins from './plugins/server';
+
+const pkgInfo = require('../package.json');
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+// default config that is used across all builds
+const config = {
+  input: pkgInfo.source || './index.js',
+  treeshake: {
+    // We assume reading a property of an object never has side-effects so we
+    // disable this option as it can significantly reduce bundle size but
+    // can potentially break functionality if you rely on getters or errors
+    // from illegal property access.
+    propertyReadSideEffects: false
+  }
+};
+
+export default function makeConfig(commandOptions) {
+  const builds = [];
+
+  // iife build (for use with dev-server)
+  const iife = {
```

Comment:
Let's maybe rename this variable to `example` so it's clear it's only the example deck (which we don't publish to npm).